templates/firefox-ios.xliff

```diff
@@ -2205,6 +2271,58 @@
         <source>General</source>
         <note>General settings section title</note>
       </trans-unit>
+      <trans-unit id="Settings.Home.Option.Description" xml:space="preserve">
+        <source>Choose content you see on the on Firefox homepage.</source>
+        <note>Description below the settings section describing what the section is for.</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.JumpBackIn" xml:space="preserve">
+        <source>Jump Back In</source>
+        <note>Option in settings to toggle Jump Back In section on homepage</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Pocket" xml:space="preserve">
+        <source>Recommended by Pocket</source>
+        <note>Option in settings to turn on off pocket recommendations</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.RecentSearches" xml:space="preserve">
+        <source>Recent Searches</source>
+        <note>Option in settings to toggle Recent Searches section on homepage</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.RecentlySaved" xml:space="preserve">
+        <source>Recently Saved</source>
+        <note>Option in settings to toggle Recently Saved section on homepage</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.RecentlyVisited" xml:space="preserve">
+        <source>Recently Visited</source>
+        <note>Option in settings to toggle Recently Visited section on homepage</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.Shortcuts" xml:space="preserve">
+        <source>Shortcuts</source>
+        <note>Option in settings to toggle Shortcuts section on homepage</note>
+      </trans-unit>
+      <trans-unit id="Settings.Home.Option.StartAtHome.AfterFourHours" xml:space="preserve">
+        <source>Homepage after four hours of inactivity</source>
+        <note>Option in settings to set start at home feature timing to After four hours</note>
```

Comment:
Nit: The way the sentence is constructed is a bit hard to understand as is.
Would recommend something like: "Option in Settings, to set to start the "home" feature timing to "After four hours""